drafthorse/models/elements.py

```diff
@@ -67,10 +67,14 @@ def append_to(self, node):
             node.append(el)
 
     def serialize(self, schema="FACTUR-X_BASIC"):
-        xml = b'<?xml version="1.0" encoding="UTF-8"?>' + ET.tostring(
```

Comment:
This concatenation issue was fixed by adding the "utf-8" parameter to tostring method.